src/Generator/Types/Std/Stdlib.cs

```diff
@@ -198,6 +194,8 @@ public uint GetCharPtrWidth(PointerType pointer)
                     return Context.TargetInfo.WCharWidth;
                 if (pointee.IsPrimitiveType(PrimitiveType.Char16))
                     return Context.TargetInfo.Char16Width;
+                if (pointee.IsPrimitiveType(PrimitiveType.Char32))
```

Comment:
We have an overload which takes an `out` parameter. Could you please use this overload and then switch on the result?